c/misra/src/rules/RULE-16-4/EverySwitchShallHaveDefaultLabel.ql

```diff
@@ -0,0 +1,32 @@
+/**
+ * @id c/misra/every-switch-shall-have-default-label
+ * @name RULE-16-4: Every switch statement shall have a default label
+ * @description The requirement for a default label is defensive programming.
```

Comment:
I believe we can be a bit more informative in the description.
Something like "A default label that has no statements or a comment explaining why this is correct indicates a missing implementation that may result in unexpected behavior when the default case is executed."